serverless/database-backup/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+PG_MAJOR_VERSION=$(echo $PG_VERSION | cut -d'.' -f1)
```

Comment:
how about
```
amazon-linux-extras enable postgresql11

yum clean metadata && yum install postgresql -y
```

Reply:
Was using this approach originally. But then came across [this issue](https://unix.stackexchange.com/questions/77083/pg-dump-version-mismatch-on-debian), which made me feel a little unsure on whether `pg_dump` is indeed compatible within a major version. Perhaps it's fixed after postgresql9, have not looked deeper into this. 

Other concern was also that postgresql12 is not available through this method. Amazon release cycle seems to be quite slow too. [This issue](https://stackoverflow.com/a/57973741) in 2019 was still asking for postgresql11 support. If there's some sort of security vulnerability that require us to bump the major version, this method might break.